test.js

```diff
@@ -33,7 +33,6 @@ describe('capture-exit', function() {
   describe('.captureExit', function() {
     afterEach(function() {
       // always restore, in case we have bugs in our code while developing
-      process.exit = originalExit;
```

Comment:
I think we should keep this one. I think we should restore using abstractions not implemented in this library. As if this library breaks, our tests will break for unrelated reasons